client/settings/fraud-protection/interfaces.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { Dispatch, SetStateAction } from 'react';
+
+export interface FraudPreventionSetting {
+	block: boolean;
+	enabled: boolean;
+	min_items?: string | number | null;
+	max_items?: string | number | null;
+	min_amount?: string | number | null;
+	max_amount?: string | number | null;
```

Comment:
Thanks for the suggestion here. We talked about it a bit during our last pairing session and we think this refactor would be better tackled as a cooldown week activity so that we can get this issue finished up so we can move on to some of the other advanced fraud protection rule changes we need to make. Those changes are waiting for this TS conversion to be merged.